package echo;

message EchoRequest
{
  required string message = 1;
};

message EchoResponse
{
  required string response = 1;
};

message DummyRequest
{
  required string message = 1;
};

message DummyResponse
{
};

service EchoService
{
  rpc Echo(EchoRequest) returns (EchoResponse);
  rpc Dummy(DummyRequest) returns (DummyResponse);
};

option cc_generic_services = true;
